src/main/java/edu/greenblitz/tobyDetermined/subsystems/telescopicArm/Extender.java

```diff
@@ -119,6 +120,10 @@ public static ExtenderState getHypotheticalState(double lengthInMeters) {
 	public static double getStaticFeedForward(double elbowAngle) {
 		return Math.sin(elbowAngle + RobotMap.TelescopicArm.Elbow.STARTING_ANGLE_RELATIVE_TO_GROUND) * RobotMap.TelescopicArm.Extender.kG;
 	}
+
+	public static double getDynamicFeedForward(double elbowAngle) {
+		return getStaticFeedForward(elbowAngle) + kV * getInstance().getVelocity()/Battery.getInstance().getCurrentVoltage();
```

Comment:
remove battery voltage same as in elbow 
and the velocity should be a parameter to preserve the static nature of this function